src/Utilities/tests/for_testing/test_checkVector.cpp

```diff
@@ -10,6 +10,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 #include "catch.hpp"
+#include "OhmmsPETE/TinyVector.h"
```

Comment:
This line is actually needed if we have an actual test for the `TinyVector` case

Reply:
There was one at some point in one of the many PR's this has appeared in,  I'll find it.